src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeployerProperties.java

```diff
@@ -116,6 +125,11 @@
 	private long taskTimeout = 30L;
 
 	/**
+	 * Flag to indicate whether env variables are fed into SPRING_APPLICATION_JSON, or individually applied.
```

Comment:
Not a big fan of the term "env variables" here. Rather, deployer calls them "application properties" and *in turn*, they may end up as ENVIRONMENT VARIABLES, or SAJ, or something else theoretically